public/v2/apps/wordpress-only.json

```diff
@@ -0,0 +1,58 @@
+{
+  "captainVersion": "2",
+  "documentation": "Taken from https://docs.docker.com/compose/wordpress/. Port mapping removed from WP as it is no longer needed",
+  "displayName": "WordPress",
```

Comment:
Change display name to WordPress - No Database or something else more descriptive.

Reply:
I already had a note at the end. but change it to make more emphasis that you need to create the DB manually.